### PR TITLE
add export of opendss results in json format

### LINKED ISSUE #28: Add export of opendss results in Json format
...to be consumed by scenario reporting.

### PR DESCRIPTION
**Pull request by kflemin**
Resolves #28

### CODE REVIEW

**Comment by kflemin** [comment]
@tarekelgindy are you ok with this change?  I added the JSON writer so that we can pull additional information into the URBANopt reporting more easily...

**Review by vtnate — APPROVED** [review]
Works with the UO cli, using the `gmt` branches Kat made in conjunction with this.

**Review by tarekelgindy — CHANGES_REQUESTED** [review]
Sorry i just saw this now!
On line 299, the json writer should be defined with:
json_writer = JSONWriter(output_path=os.path.join(self.dss_analysis, 'json_files'))

The other parameters don't exist for the json writer

File: urbanopt_ditto_reader/urbanopt_ditto_reader.py [hunk 1]
```diff
@@ -288,10 +289,16 @@ def run(self):
...
         if not os.path.exists(os.path.join(self.dss_analysis, 'results', 'Lines')):
             os.makedirs(os.path.join(self.dss_analysis, 'results', 'Lines'), exist_ok=True)
+        if not os.path.exists(os.path.join(self.dss_analysis, 'json_files')):
+            os.makedirs(os.path.join(self.dss_analysis, 'json_files'), exist_ok=True)
 
         writer = Writer(output_path=os.path.join(self.dss_analysis, 'dss_files'), split_feeders=False, split_substations=False)
         writer.write(model)
 
+        # write in JSON format as well
+        json_writer = JSONWriter(output_path=os.path.join(self.dss_analysis, 'json_files'), split_feeders=False, split_substations=False)
```
Comment (kflemin) [inline urbanopt_ditto_reader/urbanopt_ditto_reader.py]:
```suggestion
        json_writer = JSONWriter(output_path=os.path.join(self.dss_analysis, 'json_files'))
```

Comment (kflemin) [reply urbanopt_ditto_reader/urbanopt_ditto_reader.py]: @tarekelgindy, like this?

**Pull request merged**